.github/MAP_GUIDELINES.md

```diff
@@ -54,3 +54,23 @@ If these features do not work in the current codebase then your map PR will be *
 - Feel free to take inspiration from other maps, but please don't copy paste large parts of them.
 - If you use random item spawners, try using the types of them that create specific numbers of items. This way you don't risk accidentally overloading rooms with items because RNG decided it to be so.
 - Spacemas ornaments are now automatically generated as appropriate. You must only manually add the tree somewhere, specifically the ephemeral subtype.
+
+## Less critical but still important
+- **Place Directions!** They make new maps a lot easier to navigate, not just for players, but for you as well during the process. Use the donut 2/nadir floor decals (these are controversial, some admins like them sparingly, some hate them), or the wall signs like cogmap 1, or even both. The amount of directions you should place should be proportional to map size, hallway size, and mazelikeness.
```

Comment:
If this is controversial, I wouldn't include them here.

Reply:
going to remove the mentions of the floor decals and only recommend wall decals